diesel/src/pg/expression/expression_methods.rs

```diff
@@ -1241,6 +1241,66 @@ pub trait PgJsonbExpressionMethods: Expression + Sized {
     {
         Grouped(ConcatJsonb::new(self, other.as_expression()))
     }
+    /// Creates a PostgreSQL `?` expression.
+    ///
+    /// This operator checks if the right hand side string exists as a top-level key within the JSON
+    ///
+    /// # Example
```

Comment:
I think it also will be worth to copy examples from https://www.postgresql.org/docs/devel/functions-json.html